frontend/spec/support/add_to_cart.rb

```diff
@@ -1,4 +1,6 @@
 def add_to_cart(product)
+  I18n.locale = 'en'
```

Comment:
@szymoniwacz this will break locale specs, we should not be setting this in such way

Reply:
we should replace YOUR SHOPPING CART with a translation though

Reply:
@damianlegawiec When CI is running specs, `frontend/spec/controllers/spree/store_controller_spec.rb` is changing locale to :es causing `spec/features/free_shipping_promotions_spec.rb` to fail because there is no `es` translation for `t(:added_to_cart)`. I'll work on resolving that.

Reply:
@szymoniwacz so we need to add `after do` block to reset locale to `:en`

Reply:
@damianlegawiec You're right